packages/core/src/components/advanced-select/advanced-select.scss

```diff
@@ -0,0 +1,186 @@
+@use "sass:math";
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/global/mixins/set-colors.mixin" as set-colors;
+@use "~dso-toolkit/src/components/advanced-select";
+
+:host {
+  display: block;
+}
+
+@include utilities.box-sizing();
+
+.active-option {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  border: 1px solid advanced-select.$border-color;
+  border-radius: advanced-select.$border-radius;
+  cursor: pointer;
+  background-color: advanced-select.$bg-color;
+  line-height: inherit;
+  font-family: inherit;
+  font-size: inherit;
+
+  &.open {
+    border-radius: advanced-select.$border-radius advanced-select.$border-radius 0 0;
+  }
+
+  &-label {
+    padding: advanced-select.$vertical-padding advanced-select.$horizontal-padding;
+    overflow: hidden;
+    /* stylelint-disable-next-line declaration-property-value-disallowed-list -- Nodig om lange titels op de zelfde regel te houden */
+    white-space: nowrap;
+    /* stylelint-disable-next-line declaration-property-value-disallowed-list -- Nodig om lange titels op de zelfde regel te houden */
+    text-overflow: ellipsis;
```

Comment:
We hebben hier een variant van het label voor: "truncate". die toont de afgekapte content op hover/focus.

Reply:
Is het component `dso-label` hier wel gepast? Ik krijg dit niet mooi verwerkt binnen dit component

Reply:
Hmm, dat zou dan een makke van het component zijn. Maandag even naar kijken